src/python/ensembl/io/genomio/assembly/download.py

```diff
@@ -281,6 +282,8 @@ def main() -> None:
     parser.add_argument_dst_path(
         "--download_dir", default=Path.cwd(), help="Folder where the data will be downloaded"
     )
+    parser.add_log_arguments()
```

Comment:
Not quite sure where add_log_arguments is inherited from

Reply:
I assume from a future update to ensembl-py's ArgumentParser (so we can't use that PR right now)

Reply:
That was my guess yep

Reply:
That's correct. I do not want to do any PRs into `ensembl-py` until we like what we have here.